shims/spark311/src/main/scala/com/nvidia/spark/rapids/shims/spark311/ParquetCachedBatchSerializer.scala

```diff
@@ -937,6 +956,31 @@ class ParquetCachedBatchSerializer extends CachedBatchSerializer with Arm {
     }
   }
 
+  private def getStatsFromBlockMeta(blockmetas: List[BlockMetaData]): InternalRow = {
+    case class Holder(var count: Int = 0, var size: Long = 0)
+    // assuming all columns are in the same order within all blocks and have the same number of cols
+    val stats = blockmetas(0).getColumns.asScala.map { c =>
+      (Statistics.createStats(c.getPrimitiveType), Holder())
+    }
+    val cols = blockmetas(0).getColumns.size()
+    var j = 0
+    while (j < cols) {
```

Comment:
nit: it looks like these two `while` loops could be simplified to be `for` loops over a range, such as `for (j <- 0 until cols)`?